src/Child.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace Antidot\React;
+
+class Child
+{
+    public static function fork(int $numberOfWorkers, callable $asyncServer, int $numberOfFork = 0): void
+    {
+        $pid = pcntl_fork();
```

Comment:
Just notice that the `pcntl_fork` function is only available on Unix-based system.

The reference is available [here](https://www.php.net/manual/en/intro.pcntl.php).

Reply:
I am searching for an alternative for windows without luck, this feature will only be available on nix systems I think. We can protect it from usage in Windows systems;-S

Reply:
something like this:
![image](https://user-images.githubusercontent.com/1093654/105590999-82713e00-5d92-11eb-926d-521cce618be4.png)

Reply:
I'm working o mutation tests now;- D

Reply:
BTW, do you consider using the `pthread`?

Reply:
Thanks for the suggestion @peter279k :smiley::smiley::smiley:. I'll check it now. But for sure, I would like to allow using all features in any OS.

Reply:
pthread is discontinued from PHP8.0
Instead of pthreads, there is a new package from the same mantainer named Parallel. It's an extension and right now only work with PHP7. I tried it, and TBH, I did not make work at all.

Reply:
@peter279k @mmoreram I've tried parallel extension with @xserrat a time ago, and we discard it for complexity issues(we make it run with a docker image created by @WyriHaximus), both pthreads and parallel requires PHP compiled with ZTS and extra configuration(enable extension). 
I propose to maintain this feature without the windows multi-thread support and opening a new issue to research and then add multi-thread support in windows systems.

What do you think?

Reply:
Check [this](https://www.php.net/manual/en/book.parallel.php). The `Parallel` extension looks good :).

Reply:
@kpicaza In the short term that's the easiest way to make this happen. Really interested to see how well this works with different event loops as this isn't something we've tried in the past.

Reply:
@peter279k Yeah, I've build https://github.com/reactphp-parallel around it for a reason 😉 .

Reply:
Thanks @WyriHaximus, @mmoreram, and @peter279k for guidance;- D. I've just opened a new issue https://github.com/antidot-framework/react-framework/issues/14 to add support and document the usage by parallel ext.